### PR TITLE
Added instructions for LXC/LXD containers

### PR DESCRIPTION
**Pull request by Kentoseth**
- Added an install-with-lxd.md file
- Modified the index.rst file to include reference to this document

### CODE REVIEW

**Comment by ttmc** [comment]
Hi @Kentoseth 

Before we can merge this pull request, which may contain your intellectual property in the form of copyright or patents, our lawyers say we need you or your organization to agree to one of our contributor agreements. If you are contributing on behalf of yourself (and not on behalf of your employer or another organization you are part of) then you should:

1. Go to: https://www.bigchaindb.com/cla/
2. Read the Individual Contributor Agreement
3. Fill in the form "For Individuals"
4. Check the box to agree
5. Click the SEND button

If you're contributing as an employee, and/or you want all employees of your employing organization to be covered by our contributor agreement, then someone in your organization with the authority to enter agreements on behalf of all employees must do the following:

1. Go to: https://www.bigchaindb.com/cla/
2. Read the Entity Contributor Agreement
3. Fill in the form "For Organizations”
4. Check the box to agree
5. Click the SEND button

We will email you (or your employer) with further instructions.

File: docs/source/appendices/install-with-lxd.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Installing bigchainDB on LXC containers using LXD
```
Comment (ttmc) [inline docs/source/appendices/install-with-lxd.md]: This is minor point, but could you change this title to Title Case (and always capitalize BigchainDB; it's a proper noun), i.e.

```text
# Installing BigchainDB on LXC Containers Using LXD
```

File: docs/source/appendices/install-with-lxd.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Make sure your container is running by typing:
+
+`lxc list`
+
+Now, from the host (and the correct directory) where you saved `install.sh`, run this command:
+
+`cat install.sh | lxc exec bigchaindb /bin/bash`
+
+If you followed the commands correctly, you will have successfully created an LXC container (using LXD) that can get you up and running with bigchainDB in <5 minutes (depending on how long it takes to download all the packages).
```
Comment (ttmc) [inline docs/source/appendices/install-with-lxd.md]: As earlier, please capitalize BigchainDB.

**Comment by Kentoseth** [comment]
I was told to add this to the comments:

`8mNpVQLUaDTLCbH8I6VaOQ4RH2bOTacGgkBgaenakFAsIeYWhVPMx6Plzorn9eF`

**Comment by Kentoseth** [comment]
Please confirm if everything is correct.

Thanks,

**Comment by Kentoseth** [comment]
BEGIN BLOCK

This is to confirm that I agreed to and accepted the BigchainDB Individual Contributor Agreement at https://www.bigchaindb.com/cla/ and to represent and warrant that I have authority to do so.

8mNpVQLUaDTLCbH8I6VaOQ4RH2bOTacGgkBgaenakFAsIeYWhVPMx6Plzorn9eF

END BLOCK

**Comment by Kentoseth** [comment]
Very painful process to contribute 1 install doc :)

**Comment by ttmc** [comment]
Now that you've agreed to the CLA, you'll never have to do it again.

**Pull request merged**